dashboard/lib/mega_section.rb

```diff
@@ -60,6 +68,7 @@ def self.create_teacher(email, password, name)
         raise "Not a sample teacher - #{user.name}"
       end
       environment_check!
+      user.teacher_feedbacks.with_deleted.delete_all
       user.really_destroy!
```

Comment:
Deleting all of the user's TeacherFeedbacks was one of the slower steps in the delete process; by pre-emptively removing them in one query here we save some time in the `really_destroy!` step.